.github/workflows/unit.yml

```diff
@@ -0,0 +1,90 @@
+name: unit
+
+on:
+  push:
+    branches: [master]
+  pull_request: # run on all PRs, not just PRs to a particular branch
+
+jobs:
+  # `unit` includes just unit and proto tests.
+  unit:
+    strategy:
+      matrix:
+        node: ['12', '14', '16']
+    runs-on: ubuntu-latest
+    name: node ${{ matrix.node }}
+    env:
+      LATEST_NODE: '16'
```

Comment:
AFAICT it's not possible to refer to the `matrix.node` full array in steps nor `env` variables inside `matrix` (even with fancy `fromJSON` stuff), so this was the best I could do to make a way to keep the conditionals on latest node easy to keep up to date.